src/components/ApplyToHostBtnLoggedIn.js

```diff
@@ -84,6 +85,14 @@ class ApplyToHostBtnLoggedIn extends React.Component {
       );
     }
 
+    if (host && host.slug === 'opensource') {
+      return (
+        <Button className="blue" href={`/${host.slug}/apply`}>
+          <FormattedMessage id="host.apply.create.btn" defaultMessage="Apply to create a collective" />
+        </Button>
+      );
+    }
```

Comment:
I don't get this change. Why the need to hardcode it? Why not rely on the following code below?

```
 <div className="ApplyToHostBtnLoggedIn">
          {!this.inactiveCollective && (
            <Button className="blue" href={`/${host.slug}/apply`}>
              <FormattedMessage id="host.apply.create.btn" defaultMessage="Apply to create a collective" />
            </Button>
          )}
)}
````

Reply:
If there is a reason, the condition could have just been tweaked.